### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by swar-mukh** [lirx-js/core#7] [General]
Hello, it's a really beautiful project. However, there are quite a few glaring grammatical issues in the website (documentation). Would you be willing to accept PRs?

**Comment by lifaon74** [comment]
I thanks for your interest. English is not my native language so I will be very grateful if you want to correct my words. PRs are welcome. If you require any help, I'll be glad to answer your questions.

The dev branch is called `develop`, and to start and dev the website (the documentation), you may:

```shell
cd website
yarn install
yarn start
```

Then the documentation is located in `website/docs`.

Thanks in advance for your enthusiasm.
